app/src/main/res/values-fr/strings.xml

```diff
@@ -133,6 +134,8 @@
     <string name="category_input_settings">Paramètres d\'entrée</string>
     <string name="title_checkbox_multi_controller">Prise en charge de plusieurs contrôleurs</string>
     <string name="summary_checkbox_multi_controller">Lorsqu\'elle n\'est pas cochée, tous les contrôleurs sont regroupés</string>
+    <string name="title_checkbox_vibrate_fallback">Emuler les vibrations sur votre support</string>
+    <string name="summary_checkbox_vibrate_fallback">Tester si votre appareil émule les vibrations</string>
```

Comment:
I'm not a French speaker so I'm just using Google Translate to read it, but this translation doesn't seem to convey the meaning of the original English text.

Reply:
Yes it's true I do not really understand the sentence in English at the base
Rumble must it be translated?

Reply:
Hello, it's good I just changed the problem of translations I just saw that you have released an update at the moment hoping that my translation will be in your next update good day